From b780840d967413c7638b32e8071f65fef3825998 Mon Sep 17 00:00:00 2001
From: Mel Gorman <mgorman@suse.de>
Date: Wed, 18 Jul 2018 09:35:22 +0100
Subject: [PATCH] sched/numa: Remove redundant field -kabi

References: bnc#1101669 optimise numa balancing for fast migrate
Patch-mainline: Never, kabi

This is essentially a revert of "sched/numa: Remove redundant field".
The backport of that patch could have been avoided entirely but this makes
it obvious that the patch should be dropped later when KABI is not a concern.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 include/linux/sched.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/sched.h b/include/linux/sched.h
index 7f08767403db..ac6ace3ed744 100644
--- a/include/linux/sched.h
+++ b/include/linux/sched.h
@@ -922,6 +922,8 @@ struct task_struct {
 	u64				last_sum_exec_runtime;
 	struct callback_head		numa_work;
 
+	/* UNUSED field preserved due to KABI */
+	struct list_head		numa_entry;
 	struct numa_group		*numa_group;
 
 	/*
